test/bigtest/network/config.js

```diff
@@ -45,26 +71,36 @@ export default function config() {
     return schema.lines.find(request.params.id).attrs;
   });
 
-  this.get('/locations');
+  this.put(`${LINES_API}/:id`, (schema, request) => {
```

Comment:
the same as above about `create` and `put`, along with shortcut method)

Reply:
still active about `create` and `put`

Reply:
Updated